src/WebUI/Startup.cs

```diff
@@ -53,6 +57,29 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                     name: "default",
                     pattern: "{controller=Home}/{action=Index}/{id?}");
             });
+
+            ApplyDatabaseMigrations(app, logger);
+        }
+
+        // Applies database migrations; won't cause any changes if the database is up-to-date.
+        private void ApplyDatabaseMigrations(IApplicationBuilder app, ILogger logger)
+        {
+            using (var serviceScope = app.ApplicationServices.GetRequiredService<IServiceScopeFactory>().CreateScope())
```

Comment:
```suggestion
            using (var serviceScope = app.ApplicationServices.CreateScope())
```
This seems to be enough, at least on my machine.